rotkehlchen/assets/converters.py

```diff
@@ -1142,8 +1195,12 @@ def asset_from_iconomi(symbol: str) -> AssetWithOracles:
     symbol = symbol.upper()
     if symbol in UNSUPPORTED_ICONOMI_ASSETS:
         raise UnsupportedAsset(symbol)
-    name = ICONOMI_TO_WORLD.get(symbol, symbol)
-    return symbol_to_asset_or_token(name)
+
+    return symbol_to_asset_or_token(GlobalDBHandler().get_asset_from_exchange_name(
+        exchange=Location.ICONOMI,
+        symbol=symbol,
+        default=symbol,
+    ))
```

Comment:
in all the places where you use it the method is a static method

```suggestion
    return symbol_to_asset_or_token(GlobalDBHandler.get_asset_from_exchange_name(
        exchange=Location.ICONOMI,
        symbol=symbol,
        default=symbol,
    ))
```